Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -275,6 +276,75 @@ extension VisualInstruction.Component: Codable {
         case lane
     }
     
+    /**
+     :nodoc:
+     Kind of the guidance view that provides more context about the component guidance view that may help in visual
+     markup and display choices.
+     */
+    public enum GuidanceViewKind: String, Codable, CaseIterable {
+        
+        /**
+         Junction view. Bird’s-eye artist’s rendition view of the overhead signage and
+         preferred road lane arrow on motorways where the road bifurcates into 2 or
+         more motorway trunk roads.
+         */
+        case fork = "jct"
```

Comment:
nit: imo, we should make these doc comments more clear since the naming has changed. Some of @1ec5's suggestions provide some clarity as to what each of these `GuidanceViewKind`s mean, and it would be great to let developers know.

Reply:
It's a bit hard to provide more clarity at this point, maybe in future we'll be able to extend these docs. Current info regarding sub types was taken from private docs and unfortunately they do not have enough info either.